taxi/tests/test_views.py

```diff
@@ -0,0 +1,129 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+from taxi.models import Manufacturer, Car, Driver
+
+MANUFACTURER_LIST_URL = reverse("taxi:manufacturer-list")
+CAR_LIST_URL = reverse("taxi:car-list")
+CAR_DETAIL_URL = reverse("taxi:car-detail", kwargs={"pk": 1})
```

Comment:
Ensure that the `CAR_DETAIL_URL` and `DRIVER_DETAIL_URL` patterns match the actual implementation. The hardcoded `pk` value of 1 might not correspond to an existing object in the test database.